command/ssh/proxy.go

```diff
@@ -154,6 +158,7 @@ func makeAddress(host, port string) string {
 	}
 }
 
+// nolint:unused
```

Comment:
This was part of an experiment of an ssh client support in step, right now it's disabled, but we might want to add support for it in the future, we can certainly remove it as it will be in git, but I kind of resolve it on a different way.

I'll remove it, as I don't think it's something that we're going to use soon.